Automattic/studio
# Studio: Add initial support for Ukrainian translations

Closes https://github.com/Automattic/studio/issues/451

## Proposed Changes

This PR adds support for Ukrainian language in Studio.

## Testing Instructions



* Pull the changes from this branch
* Set the language on your OS to Ukrainian
* Start the app with `npm start`
* Confirm that the app appears on Ukrainian:

<img width="879" alt="Знімок екрана 2024-08-09 о 2 53 12 пп" src="https://github.com/user-attachments/assets/d729d030-15f1-4270-ae5d-bf4191053dc2">

## Pre-merge Checklist
